recipes/xfig/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "xfig" %}
+{% set version = "3.2.6a" %}
+{% set sha256 = "a87aeb5d424aadb84ede137291bbe8649551209a807c94f0956fc323b819267c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://sourceforge.net/projects/mcj/files/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libpng
+    - jpeg
```

Comment:
Please pin `jpeg` and `libpng` as defined [here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L71) for both build and runtime requirements. Those pinnings are `jpeg 9*` and `libpng >=1.6.28,<1.7`